scrape-youtube-channel-videos-url.py

```diff
@@ -8,20 +8,24 @@
 from selenium.webdriver.support.ui import WebDriverWait
 from selenium.webdriver.support import expected_conditions as EC
 from selenium.common.exceptions import InvalidArgumentException
-from selenium.webdriver.chrome.options import Options
 
 url = sys.argv[1]
 channelid = url.split('/')[4]
 #driver=webdriver.Firefox()
-chrome_options = Options()
-chrome_options.add_argument("--user-data-dir=chrome-data")
-driver = webdriver.Chrome('chromedriver',options=chrome_options)
+driver=webdriver.Chrome()
 driver.get(url)
 time.sleep(5)
 dt=datetime.datetime.now().strftime("%Y%m%d%H%M")
 height = driver.execute_script("return document.documentElement.scrollHeight")
 lastheight = 0
 
+consent_button_xpath = "//button[@aria-label='Reject all']"
```

Comment:
I used Firefox, Chrome and Edge to have a test on my computer, but I can't repeat the Youtube cookie pop-up window on my browsers.
If you don't have the same issue, you can comment those codes.

Reply:
I think it's because I'm in Europe.

https://techcrunch.com/2022/04/21/google-to-update-cookie-consent-banner-in-europe-following-fine/